README.md

```diff
@@ -105,57 +117,129 @@ inv -l
 ```
 
 
-## Theme
+## Theme (front-end)
 
 The front-end theme for the public facing website, is split into two parts :
 - The [Jinja](https://jinja.palletsprojects.com/en/2.11.x/) templates are located inside `udata_front/theme/templates`.
-- The [Less](http://lesscss.org/) & other sourcefiles for the are located in `theme`.
-
-In addition we have a nice litle set of CSS Utilities to quickly build front end components, inspired by bootstrap, most of its documentation
-lives in the css located in `theme/less/` and is built using [Stylemark](https://github.com/mpetrovich/stylemark), you can read the live documentation
-in `udata_front/theme/stylemark/` after building it using `npm run build-stylemark`.
-
-When building pages, here are a few templates to look out for in `udata_front/theme/gouvfr/templates` :
-- `home.html` : well, duh.
-- `header.html` and `footer.html` : same idea.
-- `raw.html` : contains the general html structure exposing a `body` block where we can write our page's body.
-- `base.html` : contains some extra html structure exposing a `content` block for our page's content.
-- `subnav-large.html`, `publish-action-modal.html` and `carousel.html` : **TODO**
-
-Here are our reusable components :
-- `dataset` : datasets listings used in many pages.
-- `reuse` : cards for displaying dataset reused in the real world.
-- `participez` : is the large blue callout seen on multiple pages.
-- `macros` : **TODO**
-- `svg` : contains SVG assets to be included in our pages.
-
-### TODO
-Front docs todo :
-- Parcel 2 architecture
-  - Static copy
-  - Stylemark build
-  - VueJS compiler mode
-- VanillaJS IIFE architecture
-- Vue 3 architecture
-  - Modals
-  - I18n
-  - Config plugin
-  - Components
-Back docs todo :
-- CSS/JS file inclusion
-- Static route for UI-Kit
+- The [Less](https://lesscss.org/), [Vue](https://vuejs.org) & other sourcefiles for the are located in `theme`.
```

Comment:
That's a question for @quaxsze or @abulte ! udata_gouvfr used to split the front-end templates directly in the `theme` folder and the jinja templates in `udata_front/theme/gouvfr`, maybe to prevent front-end developers from accidentally seeing a Jinja2 template and faint, and we just kept it that way.

Reply:
it seems to me more accurate to put the JS in the folder `theme/gouvfr`. If someone wants to theme `udata-front` by creating its own theme, the JS will have to be rewritten as it is strongly bound to our components.

Reply:
See https://github.com/etalab/data.gouv.fr/issues/565